test/runtests.jl

```diff
@@ -75,6 +75,20 @@ struct CubeRooter end
     @test collect(x) == [1, 2, 4]
     @test collect(x) isa Vector{Int}
 
+    x = [1, 2, missing, 4]
+    y = [1, 2, 3, missing]
+    z = [missing, missing, 3, 4]
+    mx, my = skipmissings(x, y)
+    @test eltype(mx) == Int
+    @test collect(mx) == [1, 2]
+    @test collect(mx) isa Vector{Int} 
+    mx, my, mz = skipmissings(x, y, z)
+    @test isempty(collect(mx))
+    x = [missing 4; 2 5; 3 6]
+    y = [1 4; missing 5; 3 6]
+    mx, my = skipmissings(x, y)
+    @test collect(mx) == [3, 4, 5, 6]
```

Comment:
Also test `mapreduce` (see tests for `skipmissing` in Julia, as these are tricky to cover 100%) and `getindex`. Put a few `@inferred` when it makes sense as that's really essential.

Would also be good to test `IndexStyle` and `length`/`size`.

Reply:
This still applies AFAICT?

Reply:
`mapreduce`, `getindex`, and `IndexStyle` were already covered. 

`length` and `size` are not supported so we don't test. 

I added an `@inferred` test.